Don't install headers or .pc file: they would conflict between installs for
multiple python versions. In case those are needed, something must be found to
handle name conflicts.

--- bindings/python/CMakeLists.txt.orig	2022-08-16 12:08:52.000000000 +0200
+++ bindings/python/CMakeLists.txt	2022-08-16 12:11:31.582076567 +0200
@@ -12,9 +12,6 @@
       ${CMAKE_CURRENT_SOURCE_DIR}/${header}
       ${${PROJECT_NAME}_BINARY_DIR}/include/${PROJECT_NAME}/${SUBPATH}/${header})
 
-    INSTALL(FILES ${CMAKE_CURRENT_SOURCE_DIR}/${header}
-      DESTINATION ${CMAKE_INSTALL_PREFIX}/include/${PROJECT_NAME}/${SUBPATH}/${headerPath}
-            PERMISSIONS OWNER_READ GROUP_READ WORLD_READ OWNER_WRITE)
   ENDFOREACH(header)
 ENDMACRO(SYMLINK_AND_INSTALL_HEADERS HEADERS SUBPATH)
 
@@ -205,7 +202,6 @@
   MAKE_DIRECTORY("${${PROJECT_NAME}_BINARY_DIR}/include/pinocchio/bindings/python/algorithm/parallel")
 ENDIF(BUILD_WITH_OPENMP_SUPPORT)
 MAKE_DIRECTORY("${${PROJECT_NAME}_BINARY_DIR}/include/pinocchio/bindings/python/utils")
-SYMLINK_AND_INSTALL_HEADERS("${${PROJECT_NAME}_PYTHON_HEADERS}" "bindings/python")
 
 # Headers of the Python parser
 IF(BUILD_WITH_LIBPYTHON)
@@ -349,11 +345,6 @@
      "${CMAKE_CURRENT_SOURCE_DIR}/pinocchiopy.pc.cmake"
      "${CMAKE_CURRENT_BINARY_DIR}/pinocchiopy.pc")
 
-  INSTALL(
-      FILES "${CMAKE_CURRENT_BINARY_DIR}/pinocchiopy.pc"
-      DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig
-      PERMISSIONS OWNER_READ GROUP_READ WORLD_READ OWNER_WRITE)
-
   IF(DOXYGEN_FOUND AND DOXYGEN_VERSION VERSION_GREATER 1.8.17)
     SET(DOXYGEN_GENERATE_HTML YES)
     SET(DOXYGEN_GENERATE_LATEX NO)
